config/finance/2024/Expenses.yml

```diff
@@ -95,4 +95,28 @@ August:
   - Category: 3rd Party Services
     Amount: '1354.35'
   - Category: AsyncAPI Conf on Tour 2024
-    Amount: '1384.70'
\ No newline at end of file
+    Amount: '1384.70'
+September:
+  - Category: Bounty Program
+    Amount: '3000.00'
```

Comment:
don't worry about a spike, it is not a mistake, it's just that Q4 bounty got much more visibility from maintainers (plus some Q3 payouts), I think the amount will increase with each bounty Q so more and more maintainers use it regularly to get some funding or get some work on the project done.

cc @aeworxet